content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -125,6 +125,10 @@ containers. Cilium is L7/HTTP aware and can enforce network policies on L3-L7
 using an identity based security model that is decoupled from network
 addressing, and it can be used in combination with other CNI plugins.
 
+### Hybridnet
+
+[Hybridnet](https://github.com/alibaba/hybridnet) is an open source CNI plugin designed for hybrid clouds which provides both overlay and underlay networking for containers in one or more clusters. Overlay and underlay containers can run on the same node and have cluster-wide bidirectional network connectivity. Hybridnet always focus on productive large-scale, user-friendliness and heterogeneous infrastructure.
```

Comment:
Please remote the adjectives that smell like marketing materials.

Reply:
Thanks, those words have be removed.